_inc/lib/debugger/jetpack-debugger-site-health.js

```diff
@@ -63,7 +63,9 @@ jQuery( document ).ready( function( $ ) {
 	};
 
 	if ( jetpackSiteHealth.progressPercent ) {
-		JetpackSync.init();
+		setTimeout( function() {
```

Comment:
Note: Timeout is needed as site health output is added after the document is ready.